doc/source/examples/multiphysics/concentric-heat-exchanger/concentric-heat-exchanger.rst

```diff
@@ -0,0 +1,228 @@
+        subsection bc 2
+            set id = 2
+            set type              = noflux
+        end
+    
+        subsection bc 3
+            set id = 3
+            set type              = temperature
+    	      set value             = 0
+        end
+    
+        subsection bc 4
+            set id = 4
+            set type              = noflux
+        end
+    end
+
+
+Next, we define the physical properties for both the solid and the fluid. It is important to explicitly indicate the number of solids, otherwise, the solid region will not be detected by Lethe. We consider the physical properties of copper for the solid and water for the fluid. The exponent that arises results from the change of units for length from meter to millimeter. 
+
+.. code-block:: text
+
+   subsection physical properties
+     set number of solids = 1
+     subsection fluid 0
+       set kinematic viscosity            = 1
+       set specific heat                  = 4180e6
+       set density                        = 1000e-9
+       set thermal conductivity           = 0.60e3
+     end
+   
+       subsection solid 0
+       set thermal conductivity           = 398e3
+       set specific heat                  = 385e6
+       set density                        = 8960e-9
+     end
+   end
+
+To enable a more complete analysis of the case, we enable the heat flux post-processing. This will calculate the total heat flux on every boundary of the domain and enable us to characterize the energy coming in and out of every inlet and outlet.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Post-processing
+    #---------------------------------------------------
+    subsection post-processing
+        set verbosity = verbose
+        set calculate heat flux = true
+    end
+
+Finally, we are interested in steady-state results and we thus specify a steady-state simulation.
+
+.. code-block:: text
+
+    # --------------------------------------------------
+    # Simulation Control
+    #---------------------------------------------------
+    subsection simulation control
+      set method                  = steady
+      set output frequency        = 1
+      set output path             = ./output/
+    end
+
+
+
+-------
+Results
+-------
+
+
+The following image shows the temperature profile along the length of the inner tube for three radial positions: center(:math:`r=0mm`), half radius (:math:`r=0.5mm`) and inner wall (:math:`r=1mm`). We see that the temperature at the center of the tube takes a certain length before it starts decreasing. This is due to the poor heat transfer within the liquid. The black circle indicates the outlet temperature calculated from the NTU approach using correlation. We see that this temperature is well within the envelope of the temperature profile obtained at the outlet. 
+
+ .. image:: images/temperature_along_line.png
+    :alt: problem_illustration
+    :align: center
+    :width: 300
```

Comment:
Increase size of image because it is not readable and adjust indentation:

```suggestion
.. image:: images/temperature_along_line.png
    :alt: problem_illustration
    :align: center
    :width: 500
```